src/main/java/roomescape/presentation/web/request/CreateReservationTimeWebRequest.java

```diff
@@ -0,0 +1,16 @@
+package roomescape.presentation.web.request;
+
+import java.time.LocalTime;
+import jakarta.validation.constraints.NotNull;
+import roomescape.application.request.CreateReservationTimeRequest;
+
+public record CreateReservationTimeWebRequest(
+
+        @NotNull
+        LocalTime startAt
+) {
```

Comment:
```suggestion
public record CreateReservationTimeWebRequest(@NotNull LocalTime startAt) {
```
() 안에서는 어노테이션은 한줄로 처리해주는게 제눈에는 안어색해보이네요!
회사마다 컨벤션은 다를 수 있지만 제 경험상 이렇게 처리 했던적은 없어서요

Reply:
이런 경우에는 한줄로 처리해주는게 더욱 자연스러워 보이네요! `NotNull LocalTime`이다 처럼 자연스럽게 읽히는 것 같아요! 
현재는 과하게 줄바꿈 되어 있는 부분을 변경했습니다. 감사합니다. 😀

Reply:
++ 추가적으로 컨벤션이 궁금해서, spring-data-jpa 프로젝트의 구현체를 봤는데요. `@Autowired` 같은 경우에도 제안주신 스타일대로 작성하더라구요. 영이가 경험하셨던 컨벤션에서는 너무긴 어노테이션이 아니라면, 모두 일렬로 나열하시는 편이었는지 궁금합니다!

Reply:
질문을 주신 케이스가 명확하게 어떤 상황인지는 모르겠지만
파라미터의 경우는 어노테이션을 일렬로 관리하고
필드의 경우에는 위쪽으로 붙이는 방식을 사용하는 편입니다

컨벤션의 영역은 회사마다 다를 수 있어서 저는 이렇게 쓴다 정도로 참고해주시면 좋을것 같네요!

Reply:
Record 클래스는 파라미터에 선언된 친구들을 값으로 가지니, 일반 클래스에서 적용되는 컨벤션이랑 헷갈렸네요! (개행도 포함해서요.)
확실히 컨벤션의 영역이긴 하지만, 만약 팀 컨벤션이 없다면, 영이가 제안주신 컨벤션이 마음에 들어서 앞으로 파라미터는 일렬, 필드는 위에다 붙이는 방식을 사용할 것 같아요! 감사합니다. 😀